cartoframes/auth/__init__.py

```diff
@@ -20,6 +20,14 @@ def set_default_context(context=None, base_url=None, api_key=None, creds=None, s
             api_key='your api key'
         )
 
+    From a base_url (for public datasets)
```

Comment:
I like the idea of educating the user about what's happening. How about saying something about how `default_public` is used for the API key in this case?

Reply:
:+1: